assets/script.js

```diff
@@ -82,4 +82,20 @@ if (document.location.pathname.indexOf('fullcpgrid') > -1) {
 // Current year of footer
 const currentYearSpan = document.getElementById('currentYear')
 const currentYear = new Date().getFullYear()
-currentYearSpan.innerText = currentYear
\ No newline at end of file
+currentYearSpan.innerText = currentYear
+
+// Search bar 
+function search_card() {
+  let input = document.getElementById('searchbar').value
+  input=input.toLowerCase();
+  let x = document.getElementsByClassName('card');
```

Comment:
Again, naming is very important. `x`does not tell us anything about what this variable holds. It makes it hard for someone looking at the code for the first time to know what it is, especially later on the code where we are using it.  
To this end, this variable is actually holding all the cards in the DOM so i would simply call it `cards` and use that everywhere we had `x` before.